test/throw.test.js

```diff
@@ -20,12 +23,9 @@ test('Fastify should throw on multiple assignment to the same route', t => {
 
   fastify.get('/', () => {})
 
-  try {
+  t.throws(() => {
     fastify.get('/', () => {})
-    t.fail('Should throw on duplicated route declaration')
-  } catch (error) {
-    t.equal(error.message, "Method 'GET' already declared for route '/'")
-  }
+  }, new FST_ERR_DUPLICATED_ROUTE('GET', '/'))
```

Comment:
Instead of matching the error.
Check for `error.code` property, `message` often changed but not consider as breaking change.

Reply:
will do.

Reply:
done...